styles.css

```diff
@@ -100,7 +100,7 @@ main {
   background-color: #ebf0ee;
 }
 
```

Comment:
Please ensure that the page has a minimum contrast ratio of at least 4.5:1 for normal-size text. Below is the test result, some of the tests are failing. Kindly look into this.

https://www.aditus.io/button-contrast-checker/rega1237-github-io-2022-03-29-at-21-15-53-167?failing